docs/components/HomePageCard.js

```diff
@@ -0,0 +1,52 @@
+// @flow strict
+import { Box, Card, Flex, Heading, Mask, TapArea, Text } from 'gestalt';
+import type { Node } from 'react';
+
+type Props = {|
+  image?: string,
+  description: string,
+  title: string,
+  color: string,
+  href: string,
+|};
+
+const HomePageCard = ({ image, description, title, color, href }: Props): Node => (
+  <Mask rounding={4}>
```

Comment:
@ayeshakmaz Looks like the `Mask` wrapper component breaks the focus styles on the `HomePageCard` component:

# Before

![image](https://user-images.githubusercontent.com/127199/149362974-a4d087d2-b6b3-4788-ada7-d46635e9198b.png)


# After

![image](https://user-images.githubusercontent.com/127199/149362908-91d2abbf-3538-4dd7-9cfe-57ec2bbc84ba.png)

We can probably fix this by setting the border radius inline